STACK-UP-3/warriorz-backend
# #171784074 Configure ESLint

#### What does this PR do?
 Having ESLint configured.

#### Description of Task to be completed?
  Having ESLint configured to help check for style violations and flag them.
  Having ESLint configured to use Airbnb convention

#### How should this be manually tested?
After cloning the repo, cd into it, create a <name>.js file
and then try different codes like importing module which does exist and so on.
You will notice warning or error.

#### What are the relevant pivotal tracker stories?
[#171784074](https://www.pivotaltracker.com/n/projects/2438874/stories/171784074)
